StaryNight/src/main/java/com.starynight/service/SavesService.java

```diff
@@ -1,23 +1,32 @@
 package com.starynight.service;
 
 import com.starynight.dto.SaveData;
+import com.starynight.dao.ISavesDAO;
 import org.springframework.stereotype.Service;
 
 import java.util.ArrayList;
 import java.util.List;
 
 @Service
 public class SavesService implements ISavesService {
+    @Autowired
```

Comment:
Rewrote the service class to use the DAO, autowiring the savesDAO variable to the DAOStub